src/v2/ion/RemediationConstants.js

```diff
@@ -133,6 +134,9 @@ const TERMINAL_FORMS = [
   FORMS.TERMINAL,
   FORMS.DEVICE_ENROLLMENT_TERMINAL,
 ];
+const IDP_FORM_TYPE = {
```

Comment:
what are other `IDP form types` ?

Reply:
`GOOGLE` `FACEBOOK` `LINKEDIN` etc. do you want me to add everything here?

Reply:
Nope.